examples/timestamp/package.json

```diff
@@ -13,24 +13,24 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^17.0.0-next.0",
-    "@angular/common": "^17.0.0-next.0",
-    "@angular/compiler": "^17.0.0-next.0",
-    "@angular/core": "^17.0.0-next.0",
-    "@angular/forms": "^17.0.0-next.0",
-    "@angular/platform-browser": "^17.0.0-next.0",
-    "@angular/platform-browser-dynamic": "^17.0.0-next.0",
-    "@angular/router": "^17.0.0-next.0",
+    "@angular/animations": "^17.0.",
```

Comment:
Guessing you didn't meant to lop off the 0s at the end...

Reply:
Good catch, that was by mistake. Resolved with [ec1380a](https://github.com/just-jeb/angular-builders/pull/1515/commits/ec1380a2f883baeea2c12e8af164d0b172eab992)